src/CreateDisc.js

```diff
@@ -0,0 +1,121 @@
+        }
+    };
+    render() { 
+        return (
+            <>
+                {this.state.message && (
+                    <div 
+                        className={classNames({
+                            message: true,
+                            error: this.state.error,
+                            success: !this.state.error,
+                        })}
+                    >{this.state.message}</div>
+                )}
+                <h1>{this.state.disc}</h1>
+                <form id='create-disc'>
+                    <div className='form-group'>
+                        <label>Disc: </label>
+                        <input
+                            onChange={(e) => {
+                                this.setState({ disc: e.target.value });
+                            }}
+                            type='text'
+                            value={this.state.disc}
+                        ></input>
+                    </div>
+                    <div className='form-group'>
+                        <label>Speed: </label>
+                        <input
+                            onChange={(e) => {
+                                this.setState({ speed: e.target.value });
+                            }}
+                            type='text'
+                            value={this.state.speed}
+                        ></input>
+                    </div>
+                    <div className='form-group'>
+                        <label>Type: </label>
+                        <input
+                            onChange={(e) => {
+                                this.setState({ type: e.target.value });
+                            }}
+                            type='text'
+                            value={this.state.type}
+                        ></input>
+                    </div>
+                    <div className='form-group'>
+                        <label>Stable: </label>
+                        <select 
+                            value={this.state.stable}
+                            onChange={(e) => {
+                                this.setState({ stable: e.target.value });
+                            }}    
+                        >
+                                <option>true</option>
+                                <option>false</option>
+                        </select>
+                    </div>
+                    <div className='form-group'>
+                        <label>Manufacturer: </label>
+                        <select 
+                            value={this.state.manufacturer}
+                            onChange={(e) => {
+                                this.setState({ manufacturer: e.target.value });
+                            }}
+                        >
+                            {this.state.manufacturers.map((mnfr) => {
+                                return (
+                                    <option key={mnfr.name} value={mnfr.name}>{mnfr.name}</option>
```

Comment:
Fun tip! Since you are only returning a single line of code here, you can use what's called an _`implicit return`_ here and that would reduce the amount of code needed to this: 

```javascript
{this.state.manufacturers.map((mnfr) => <option key={mnfr.name} value={mnfr.name}>{mnfr.name}</option>)}
```